Readme.md

```diff
@@ -96,3 +96,19 @@ you change, you may have to reload the extension in your browser:
   Reload.
 - If you changed the 'content_script', it seems browser-dependent whether newly
   opened pages will get the new version. Better reload the extension to be sure.
+
+### Guideline for Contribution
+
+Please follow the given guideline to make it easier for the maintainer to review and evaluate your work.
+
+**NOTE:**By sending a pull request you willingly waive all your copyrights on the code, so that it can be published in the public domain. For further information look into [Unlicense](http://unlicense.org/)
+
+* It is recommended that the developers use a text editor that supports editor config files for consistent spacing and other code styling.
```

Comment:
If we wanna put editor recommendations, maybe also put a note here to use an editor that supports eslint. Just as important, and relates to code styling note here. Beats running `npm run lint` every time you want to see if you've broken some consistent style and to catch certain types of bugs early.